jwst/exp_to_source/main.py

```diff
@@ -1,73 +1,71 @@
-import os
 import sys
 import argparse
+from pathlib import Path
 
 from stdatamodels.jwst.datamodels import MultiSlitModel
 
 from jwst.exp_to_source import exp_to_source
 
 
-class Main():
-    """Convert exposure-based slits data to source-based data
-    Docs from the source.
-
-    Parameters
-    ----------
-    args: str or [str,...]
-        The command-line arguments. This is passed to
-        `argparse.parse_args`. If None, `sys.argv`
-        is used.
+class Main:
+    """
+    Convert exposure-based slits data to source-based data.
 
-    Attributes
-    ----------
-    sources: {source: MultiExposureModel, ...}
-        A dict keyed on source name whose value is
-        the corresponding MultiExposureModel.
+    Docs from the source.
     """
 
     def __init__(self, args=None):
+        """
+        Full documentation from the source.
 
+        Parameters
+        ----------
+        args : str or [str,...]
+            The command-line arguments. This is passed to
+            `argparse.parse_args`. If None, `sys.argv`
+            is used.
+        """
         if args is None:
             args = sys.argv[1:]
         if isinstance(args, str):
-            args = args.split(' ')
+            args = args.split(" ")
 
         parser = argparse.ArgumentParser(
-            description='Convert exposure-based data to source-based data',
-            usage='python -m jwst.exp_to_source.main files'
+            description="Convert exposure-based data to source-based data",
+            usage="python -m jwst.exp_to_source.main files",
         )
-        parser.add_argument(
-            'files',
-            type=str,
-            nargs='+',
-            help='Files to convert')
+        parser.add_argument("files", type=str, nargs="+", help="Files to convert")
 
         parser.add_argument(
-            '-o', '--output-path',
+            "-o",
+            "--output-path",
             type=str,
-            default='.',
-            help='Folder to save results in. Default: "%(default)s"'
+            default=".",
+            help='Folder to save results in. Default: "%(default)s"',
         )
 
         parser.add_argument(
-            '--dry-run',
-            action='store_true', dest='dry_run',
-            help='Execute but do not save results.'
+            "--dry-run",
+            action="store_true",
+            dest="dry_run",
+            help="Execute but do not save results.",
         )
 
         try:
             parsed = parser.parse_args(args=args)
         except SystemExit:
             return
 
+        # sources : {source: MultiExposureModel, ...}
+        #    A dict keyed on source name whose value is the corresponding MultiExposureModel.
```

Comment:
remove commented lines

Reply:
I left this comment on purpose to explain the variable below. I don't think we should remove it for clarity and debugging.

Reply:
```suggestion
        # self.sources is a dict keyed on source name whose value is the corresponding MultiExposureModel, e.g.
        # {source_name: MultiExposureModel, ...}
```
Gotcha, I missed that this wasn't actually code.  In that case I suggest changing the comment to look more like the above